e2e_test/iceberg/test_case/append_only_with_checkpoint_interval.slt

```diff
@@ -37,7 +37,6 @@ CREATE SINK sink1 AS select * from mv1 WITH (
     s3.region = 'us-east-1',
     s3.access.key = 'hummockadmin',
     s3.secret.key = 'hummockadmin',
-    commit_checkpoint_interval = 1
```

Comment:
What is this test case use for?  The test case name is  "...with_checkpoint_interval".  I thought it was used to test the case with interval > 1.